### DISCUSSION TITLE
Web App or Mobile App

### DISCUSSION THREAD

**Discussion by AngelUPR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#10] [Q&A]
This project is referred as being an app but I am unsure as whether it will be a web app or a mobile app

**Comment by Mannyrodd101** [comment]
I have this same question as well as I've seen some refer to it as an app and others as a website.

**Comment by AngelUPR** [comment]
I think for a better user experience, we should make this as a mobile app

**Comment by uccinimi25** [comment]
Typically cheaper to develop than [Bitlife Mod APK](https://blifeproapk.com/)  mobile apps, especially if you target multiple platforms.
Cons:
